### PR TITLE
[#IOCOM-1283] Added release and code_review actions

### PR DESCRIPTION
**Pull request by michaeldisaro**
#### List of Changes
Added release and code review actions.

#### Motivation and Context
We need to migrate also release and code review to github actions.

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: .github/workflows/code-review.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+name: "Code Review"
+
+on:
+  push:
```
Comment (Krusty93) [inline .github/workflows/code-review.yml:4]: Do we want to run the code review after merging PRs?

Comment (michaeldisaro) [reply .github/workflows/code-review.yml:4]: I found a template that did this and I think it is a good idea to get a code review also on master branch after merge. It's not really necessary, but sometimes it happened that some changes went to master and broke it.

File: .github/workflows/code-review.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    branches: ["main"]
+  pull_request:
+    types: [opened, synchronize]
+  workflow_dispatch: {}
+
+jobs:
```
[the remark this answers is not in the document]
Comment (michaeldisaro) [reply .github/workflows/code-review.yml:10]: Integration tests are very long, running them in parallel save us time.

[the remark this answers is not in the document]
Comment (Krusty93) [reply .github/workflows/code-review.yml:10]: It makes sense

File: .github/workflows/code-review.yml (hunk 1 above)
Comment (Krusty93) [reply .github/workflows/code-review.yml:4]: You're right but usually the review is done before merging to master in order to *never* break it

File: .github/workflows/code-review.yml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+          yarn install --frozen-lockfile
+          yarn start
+        shell: bash
+
+      - name: Run Integration Tests
+        run: |
+          cd __integrations__
+          yarn install --frozen-lockfile
+          sleep 30s
+          docker exec fn-service-messages-testagent yarn test
```
Comment (Garma00) [inline .github/workflows/code-review.yml:65]: Is this sleep needed?

File: .github/workflows/code-review.yml [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
```
Comment (Garma00) [inline .github/workflows/code-review.yml:94]: Ditto

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+  # Trigger the workflow on push on the main branch
+  push:
+    branches:
+      - main
```
Comment (Garma00) [inline .github/workflows/release.yml:7]: We use master in this repo

**Review by Garma00 — CHANGES_REQUESTED** [review]

File: .github/workflows/code-review.yml (hunk 1 above)
Comment (michaeldisaro) [reply .github/workflows/code-review.yml]: Yeah, it was just a nitpicky check. I removed it.

File: .github/workflows/release.yml (hunk 5 above)
Comment (michaeldisaro) [reply .github/workflows/release.yml]: I removed autorelease on push.

**Review by Garma00 — APPROVED** [review]

**Pull request merged**